src/sql/workbench/contrib/notebook/browser/notebookStyles.ts

```diff
@@ -181,27 +194,27 @@ export function registerNotebookThemes(overrideEditorThemeSetting: boolean, conf
 			collector.addRule(`
 				.notebookEditor .notebook-cell.active code-component {
 					border-color: ${inactiveBorder};
-					border-width: 0px 0px 1px 0px;
+					border-width: 1px 0px 1px 0px;
 					border-style: solid;
 					border-radius: 0;
 				}
 			`);
 
-			if (addBorderToInactiveCodeCells) {
-				// Sets a border for the editor component if we don't have a custom line color for editor instead
-				collector.addRule(`
-					.notebookEditor .notebook-cell code-component {
-						border-color: ${inactiveBorder};
-						border-width: 1px;
-						border-style: solid;
-						border-radius: 3px 3px 3px 3px;
-					}
-					.notebookEditor .notebook-cell:hover code-component {
-						border-width: 0px 0px 1px 0px;
-						border-radius: 0px;
-					}
-				`);
-			}
+			// if (addBorderToInactiveCodeCells) {
+			// 	// Sets a border for the editor component if we don't have a custom line color for editor instead
```

Comment:
If these are going to be removed, let's just delete this.

Reply:
Done